### PR TITLE
Logging improvements

### PR DESCRIPTION
**Pull request by Matthias247**
This contains 2 small logging changes which should aid debugging

### CODE REVIEW

File: quinn-proto/src/connection/mod.rs [hunk 1]
```diff
@@ -2144,6 +2143,19 @@ where
...
+            // the log with payload data
+            match &frame {
+                Frame::Crypto(f) => {
+                    trace!(offset = f.offset, len = f.data.len(), "got crypto frame");
+                }
+                Frame::Stream(f) => {
+                    trace!(id = %f.id, offset = f.offset, len = f.data.len(), fin = f.fin, "got stream frame");
+                }
+                f => {
+                    trace!("got frame {:?}", f);
```
Comment (Ralith) [inline quinn-proto/src/connection/mod.rs:2156]:
```suggestion
                    trace!("got frame");
```
The frame type is already recorded in the span.

Comment (Ralith) [reply quinn-proto/src/connection/mod.rs:2156]: Or are you intentionally including the frame payload?

Comment (Matthias247) [reply quinn-proto/src/connection/mod.rs:2156]: Yes. For all the `MAX_` frames it is interesting to know what the limits are. The payloads should never be too long apart for CRYPTO and STREAM, which are special cased for this reason

Comment (Matthias247) [reply quinn-proto/src/connection/mod.rs:2156]: Ok, guess we should special case `DATAGRAM` too

**Review by Ralith — DISMISSED** [review]

**Review by Ralith — APPROVED** [review]

**Pull request merged**